tests/docker/nginx-fpm/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+
+LOGS_PHP=(/tmp/appsec.log /tmp/helper.log /tmp/php_error.log /tmp/php_fpm_error.log)
+
+LOGS_NGINX=(/var/log/nginx/{access.log,error.log})
+touch "${LOGS_NGINX[@]}"
+chown root:adm "${LOGS_NGINX[@]}"
```

Comment:
Have you verified that the logs are available during the run? Sometimes doing this causes permission issues

Reply:
Yes, it works as it is